main.go

```diff
@@ -114,35 +121,11 @@ func main() {
 
 	go signalHandler(signalChan, doneChan)
 
-	// Periodically read number of docs and log to output
-	go func() {
```

Comment:
Should this at least be a metric instead of removing altogether?

Reply:
oh 🤦 I didn't mean to remove it - just my local changes to avoid issuing querying while ingesting. I will add an option to enable/disable it instead